src/aks-preview/HISTORY.rst

```diff
@@ -15,6 +15,7 @@ Pending
 
 0.5.139
 * `az aks create` and `az aks nodepool add`: Add warning message when specifying `--os-sku` to `Mariner` or `CBLMariner`.
+* `az aks create` and `az aks enable-addons`: Change the default value of `--enable-msi-auth-for-monitoring` to `true` and add check for airgap clouds
```

Comment:
update to this include **monitoring**  since this only applicable for monitoring addon

Reply:
changed